docs/debug.md

```diff
@@ -136,25 +136,14 @@ The syslog is particularly useful since kernel logs, launcher output, service
 status, system logs and confinement violations (these are covered in depth
 elsewhere) all get logged there.
 
-Further, systemd has log dump facility to look at all system and snap
-services. The "snappy service" command provides an easy way to monitor and
-inspect snap services. To get the status of all snap services run "snappy
-service status". To inspect the logs run "snappy service logs". Both commands
-can take a snap name to limit to a specific snap (e.g. for the service
-in `shout.sergiusens` run: `sudo snappy service logs shout`).
-
 
 ## Debugging binaries
 
 ### Installing binaries
-In order to test a new snap on a Snappy system you need to install it first.
-This is called sideloading and it can be done via
-`snappy-remote --url=ssh://ubuntu@webdm.local/ install snapname.snap`.
-Alternatively the snap can be copied via scp into the Snappy system and
-installed via `sudo snappy install snapname.snap`.
+In order to test a new snap on a Snap-based system you need to install it first.
+This is called "sideloading" and it can be done with:
```

Comment:
Installing a snap is not called sideloading, installing a snap from a local storage/store is ;-)